attributes/mysql-connectors-community.rb

```diff
@@ -1,5 +1,5 @@
 default['yum']['mysql-connectors-community']['repositoryid'] = 'mysql-connectors-community'
-default['yum']['mysql-connectors-community']['gpgkey'] = 'https://raw.githubusercontent.com/someara/yum-mysql-community/master/files/default/mysql_pubkey.asc'
+default['yum']['mysql-connectors-community']['gpgkey'] = 'https://raw.githubusercontent.com/chef-cookbooks/yum-mysql-community/master/files/default/mysql_pubkey.asc'
```

Comment:
why is this not https://raw.githubusercontent.com/opscode-cookbooks/yum-mysql-community/master/files/default/mysql_pubkey.asc as mentioned on the public issue?